### PR TITLE
Initial lazy-load and code-splitting

### PR DESCRIPTION
**Pull request by skhameneh**
This could probably be merged, merge if you'd like to be aggressive on testing.

Working on more additions.

### CODE REVIEW

File: src/ui/component/viewers/audioVideoViewer.jsx [hunk 1]
```diff
@@ -38,7 +42,14 @@ class AudioVideoViewer extends React.PureComponent<Props> {
...
     };
 
-    this.player = videojs(this.videoNode, videoJsOptions, () => {});
+    import(
+        /* webpackChunkName: "videojs" */
+        /* webpackMode: "lazy" */
+        /* webpackPreload: true */
+        'video.js'
+    ).then((videojs) => {
+      me.player = videojs(me.videoNode, videoJsOptions, () => {});
```
Comment (seanyesmunt) [inline src/ui/component/viewers/audioVideoViewer.jsx]: Can we just use `this.player`?

File: webpack.base.config.js [hunk 2]
```diff
@@ -12,9 +14,22 @@ const UI_ROOT = path.resolve(__dirname, 'src/ui/');
 const STATIC_ROOT = path.resolve(__dirname, 'static/');
 const DIST_ROOT = path.resolve(__dirname, 'dist/');
 
+console.log(ifProduction('production', 'development'))
```
Comment (seanyesmunt) [inline webpack.base.config.js]: Don't need

File: src/ui/component/viewers/audioVideoViewer.jsx (hunk 1 above)
Comment (seanyesmunt) [reply src/ui/component/viewers/audioVideoViewer.jsx]: I see it's added in a few places, do we need it?

File: webpack.base.config.js (hunk 2 above)
Comment (skhameneh) [reply webpack.base.config.js]: It's useful to see what's actively building

File: src/ui/component/viewers/audioVideoViewer.jsx (hunk 1 above)
Comment (skhameneh) [reply src/ui/component/viewers/audioVideoViewer.jsx]: Since there's arrow fn's, yes

Comment (seanyesmunt) [reply src/ui/component/viewers/audioVideoViewer.jsx]: Arrow fn's don't scope `this`. If it was a regular function you would need it.

Comment (seanyesmunt) [reply src/ui/component/viewers/audioVideoViewer.jsx]: We have something similar here https://github.com/lbryio/lbry-desktop/blob/master/src/ui/component/fileViewer/internal/player.jsx#L286

Comment (seanyesmunt) [reply src/ui/component/viewers/audioVideoViewer.jsx]: Seems to work ok with `this`

**Pull request merged**